assets/src/edit-story/components/canvas/canvasElementDropzone.js

```diff
@@ -49,7 +49,18 @@ function CanvasElementDropzone({ children }) {
 
   const onDropHandler = useCallback(
     (e) => {
-      if (isDragType(e, 'resource/media') && !activeDropTargetId) {
+      // Handles onDrop for shapes.
+      if (isDragType(e, 'shape') && !activeDropTargetId) {
+        const props = JSON.parse(e.dataTransfer.getData('shape'));
```

Comment:
`props` is a "reserved" word in eslint context. Rename to something else to avoid some of these issues.

Reply:
Done. Changed the name of the object.